src/wp-includes/rest-api/endpoints/class-wp-rest-menu-items-controller.php

```diff
@@ -854,7 +854,13 @@ public function get_item_schema() {
 			'format'      => 'uri',
 			'context'     => array( 'view', 'edit', 'embed' ),
 			'arg_options' => array(
-				'validate_callback' => static function ( $url ) {
+				'validate_callback' => static function ( $url , $args, $param ) {
+					$check_type = rest_validate_request_arg( $url, $args, $param );
```

Comment:
Does using the full `rest_validate_request_arg()` function provide any advantage here, considering the callback only needs to verify if `$url` is a string?
Wouldn't `is_string()` be simpler and quicker in this case?